requirements.txt

```diff
@@ -1,3 +1,4 @@
 sphinx
 sphinx_rtd_theme
 recommonmark
+sphinxcontrib-mermaid
```

Comment:
```suggestion
sphinxcontrib-mermaid
six
```

This made it build for me

Reply:
Fixed in b2c7a43f008e84ee16baddf6cf42a00189659710